src/components/sidechains/index.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import configureMockStore from 'redux-mock-store';
+
+import i18n from '../../i18n';
+import Sidechains from './index';
+
+describe('Sidechains', () => {
+  const store = configureMockStore([])({});
+  const options = {
+    context: { store, i18n },
+    childContextTypes: {
+      store: PropTypes.object.isRequired,
+      i18n: PropTypes.object.isRequired,
+    },
+  };
+
+  it('should render Sidechains', () => {
+    const props = {
+      t: () => {},
+    };
+    const wrapper = mount(
+      <Provider store={{}}>
+        <Router>
+          <Sidechains {...props} />
+        </Router>
```

Comment:
`withRouter(..)` in the component seems unnecessary as well as `<Router>` and `<Provider>` here